src/terra-dev-site/guide/Internationalization/InternationalizationIntro.a.guide.md

```diff
@@ -0,0 +1,85 @@
+# Internationalization (i18n) Intro
+
+Terra UI provides builtin translations and LTR / RTL support which makes Terra UI powered apps ready to use in several countries with ease. All apps built with Terra UI are required to set up translations, even if they only intend to support English. As we support apps built for use around the world, we want apps to be prepared for internationalization from the beginning of development. It is easier to add additional locales and translations to an app built with internationalization and translations from the beginning compared to retrofitting internalization and translations into an app as it evolves into new markets.
+
+# Getting Started
+Install [terra-base](https://www.npmjs.com/package/terra-base) and [terra-toolkit](https://www.npmjs.com/package/terra-toolkit).
+
+If you're using npm, run the following command:
+
+```
+npm install terra-base terra-toolkit
```

Comment:
Nitpick: Should you use the --save or --save-dev commands with this?

Reply:
Updated in this commit: https://github.com/cerner/terra-ui/pull/132/commits/cc0c8713331225234ca6d889d6cca8802fbc4d3c